openshift/imagestreams/jenkins-rhel.json

```diff
@@ -38,7 +38,7 @@
         },
         "from": {
           "kind": "DockerImage",
-          "name": "registry.redhat.io/openshift4/ose-jenkins:v4.0"
+          "name": "registry.redhat.io/openshift4/ose-jenkins:v4.10.0-202203141248.p0.g10e9ee3.assembly.stream"
```

Comment:
Given that this is to support our samples operator work, I'm ok using a specific build tag. Once we release the image out of CPaaS, we will undoubtedly update the image ref.

Reply:
exactly right @adambkaplan 

that said, we don't have to wait for this to merge for https://github.com/openshift/cluster-samples-operator/pull/416 to progress, but certainly we don't want to lose anything when the next samples operator library  bump happens

Reply:
while the registry-proxy.engineering.redhat.com/rh-osbs images have tested OK for me, the errata have not been cut such that those images have landed at registry.redhat.io so that is the OCP payload reference is still used.

We can leave JKNS-260 open until the new errata has been cut, or we can create a new Jira for tracking that and updating this entry.

I'll bring it up at the next scrum / office hours @adambkaplan @akram (who is on PTO so most likely no errata will happen until he gets back).